### PR TITLE
feature to report orf protein sequence

### PR DESCRIPTION
**Pull request by timbalam**
The last pull request was missing some commits. After fixing my local repository I was able to merge them. They add sequence to amino acid conversion and ensure that returned orf sequences have multiple of 3 lengths.

### CODE REVIEW

File: lib/finishm/orfs_finder.rb [hunk 1]
```diff
@@ -31,6 +34,15 @@ def validate_options(options, argv)
       visualise.validate_assembly_options(options)
   end
 
+  def add_orfs_options(optparse_object, options)
```
Comment (wwood) [inline lib/finishm/orfs_finder.rb]: Is it necessary to make this a separate method?

Also I think maybe `--orf-amino-acids` and `--orf-nucleotides` might be better names.

Can you add an option for --min-orf-length as well? I say make this default to 96 not 100 to be in line with `orfm` so the results are more directly comparable with graftM, yeh?

File: lib/finishm/orfs_finder.rb [hunk 2]
```diff
@@ -31,6 +34,15 @@ def validate_options(options, argv)
       visualise.validate_assembly_options(options)
   end
```
Comment (wwood) [inline lib/finishm/orfs_finder.rb]: be good to validate the options for output too - need one or more of fna and faa yeh?

Comment (timbalam) [inline lib/finishm/orfs_finder.rb]: I was thinking that it could use default names and output both if the user doesn't specify at least one of them?

Comment (wwood) [inline lib/finishm/orfs_finder.rb]: Ah ok I missed that. Nevermiund.

**Comment by wwood** [comment]
reckon this is ready to merge now?

**Comment by timbalam** [comment]
Yep, I think so.

**Pull request merged**